pacai/core/featureExtractors.py

```diff
@@ -29,26 +31,12 @@ def closestFood(pos, food, walls):
     worked on in the search project; here its all in one place
```

Comment:
Lets see, do we want to move closestFood to core/distance.py? Or keep this as a private class? Regardless this either needs to be moved to distance.py or change the name to _closestFood.

Reply:
I think we should keep it as a private class, in doing so I changed closestFood to _closestFood, since the method is only/should only be called by feature extractor classes.